### PR TITLE
Add new interconnection blog post

### PR DESCRIPTION
**Pull request by critzo**
This PR adds a commit to @collina 's work, providing a new date index page for the new blog post.

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/295)

### CODE REVIEW

**Comment by critzo** [comment]
@georgiamoon there's an issue with the image formatting on this post. I'm making another commit to resolve in a moment.

**Comment by critzo** [comment]
@collina @georgiamoon Made a few changes to the way the images and captions were formatted in this post, could you review and let me know if these are ok?
https://critzo.github.io/m-lab.github.io/blog/interconnection_update/

File: _posts/blog/2017-08-09-interconnection_update.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+![Verizon across Internap and Cogent in New York, January 2013 - January 2014, Download Throughput]({{ site.baseurl }}/images/blog/interconnection_update-Verizon-Internap_Cogent-NY.png){: width="838" height="260"}
```
Comment (georgiamoon) [inline _posts/blog/2017-08-09-interconnection_update.md]: Can we also put the source link as the click thru for the image? (and for all the images?)

**Comment by georgiamoon** [comment]
LGTM, with the one comment.

File: _posts/blog/2017-08-09-interconnection_update.md (hunk 1 above)
Comment (critzo) [inline _posts/blog/2017-08-09-interconnection_update.md]: Linking on the image works if the image is not inside a table. If you want them all consistently in a table with the caption in a bottom row, I can try just writing the whole table in html versus a mix of html and markdown.

**Pull request merged**